src/include/intarith.h

```diff
@@ -16,23 +16,35 @@
 #define CEPH_INTARITH_H
 
 #ifndef MIN
-# define MIN(a,b) ((a) < (b) ? (a):(b))
+#define MIN(a,b) ((a) < (b) ? (a):(b))
 #endif
 
 #ifndef MAX
-# define MAX(a,b) ((a) > (b) ? (a):(b))
+#define MAX(a,b) ((a) > (b) ? (a):(b))
 #endif
 
 #ifndef DIV_ROUND_UP
-# define DIV_ROUND_UP(n, d)  (((n) + (d) - 1) / (d))
+#define DIV_ROUND_UP(n, d)  (((n) + (d) - 1) / (d))
 #endif
 
 #ifndef ROUND_UP_TO
-# define ROUND_UP_TO(n, d) ((n)%(d) ? ((n)+(d)-(n)%(d)) : (n))
+#define ROUND_UP_TO(n, d) ((n)%(d) ? ((n)+(d)-(n)%(d)) : (n))
```

Comment:
Exactly , but  `DIV_ROUND_UP`(at line 27) already did this.

Others are all fixed.